### ISSUE TITLE
Show the bot vs human stat on the dashboard

### ISSUE THREAD

**Issue by rehanvdm** [rehanvdm/serverless-website-analytics#3]
There is a column (boolean) on each record that marks if the view was a bot or human. We can also show this on the dashboard.

**Comment by renekrisk** [comment]
Is this feature still planned? The boolean flag for bot vs human views would be extremely helpful on the dashboard, especially for filtering analytics

**Comment by rehanvdm** [comment]
Yes, but I haven't given this project the attention it needs lately. You are welcome to open a PR and give it a try
